NuCivic/dkan
# Site Managers should be able to see the POD validator CIVIC-5631

Issue: CIVIC-5631

## Description

- Update release notes. 
- Update changlog
- Update command center menu with link to POD validation 
- QA PR 86 on Open Data Schema Map

## QA Steps

1. Log in as sitemanager
2. Confirm you can see and use the `Project Open Data Validation` link under Site Configuration > Open Data Schema Mapper

## Acceptance Criteria
- Site Managers should be able to use the POD validator via a menu item.

## Required PRs
This build is using the `fix-permission-check` branch of ODSM
https://github.com/NuCivic/open_data_schema_map/pull/86. Merge that then switch the makefile to point back to master.
